resources/META-INF/plugin.xml

```diff
@@ -121,7 +121,7 @@
     <typeDeclarationProvider implementation="com.goide.editor.GoTypeDeclarationProvider"/>
     <codeInsight.parameterInfo language="go" implementationClass="com.goide.editor.GoParameterInfoHandler"/>
     <lang.smartEnterProcessor language="go" implementationClass="com.goide.editor.smart.GoSmartEnterProcessor"/>
-    <highlightUsagesHandlerFactory implementation="com.goide.highlighting.GoHighlightExitPointsHandlerFactory"/>
+    <highlightUsagesHandlerFactory implementation="com.goide.highlighting.exitpoint.GoHighlightExitPointsHandlerFactory"/>
```

Comment:
Directly resolved usages – yes. But there are another possible cases. Look at java/groovy implementations: 
<img width="663" alt="screenshot 2015-11-27 02 12 13" src="https://cloud.githubusercontent.com/assets/140920/11431586/789e525a-94ac-11e5-933d-ab1ef3f76666.png">

And base class is called HighlightingUsagesHandlerBase :)